Lib9c/Action/BattleArena.cs

```diff
@@ -206,6 +206,13 @@ public override IAccountStateDelta Execute(IActionContext context)
                     $" - ChampionshipId({roundData.ChampionshipId}) - round({roundData.Round})");
             }
 
+            if (context.BlockIndex - myArenaAvatarState.LastBattleBlockIndex < 2)
+            {
+                throw new CoolDownBlockException(
+                    $"[{nameof(BattleArena)}] LastBattleBlockIndex : {myArenaAvatarState.LastBattleBlockIndex} " +
+                    $"CurrentBlockIndex : {context.BlockIndex}");
+            }
+
```

Comment:
How about move this lines to line 174? Following the declaration of `myArenaAvatarState`.